guides/common/modules/proc_setting-minimal-data-collection.adoc

```diff
@@ -29,4 +29,37 @@ To remove existing data of a system, navigate to *Inventory* > *Systems* in the
 . In the {ProjectWebUI}, navigate to *{Insights}* > *Inventory Upload*.
 . Select the *Minimal data collection* setting from the dropdown menu under *Settings*.
 . When you select this option, {Project} also updates the corresponding setting on the *{Insights}* tab under *Administer* > *Settings*.
-. Click the *Generate and upload report* button to create a new inventory report and send it to the {RHCloud}.
\ No newline at end of file
+. Click the *Generate and upload report* button to create a new inventory report and send it to the {RHCloud}.
++
+.Example of data collected in a minimal report
+[source,json]
+----
+{
+  "report_slice_id": "de92044d-9d77-4895-83c4-a476f5020519",
+  "hosts": [
+    {
+      "account": "123456789",
+      "subscription_manager_id": "9fc621b9-08c3-4085-a749-bfed38c3052e",
+      "insights_id": "9fc621b9-08c3-4085-a749-bfed38c3052e",
+      "bios_uuid": "203F234F-58C7-4237-8DDF-A17A2838A66C",
+      "bios_vendor": "SeaBIOS",
+      "bios_version": "1.16.1-1.el9",
+      "arch": "x86_64",
+      "infrastructure_type": "virtual",
+      "system_profile": {
+        "installed_products": [
+          {
+            "name": "Red Hat Enterprise Linux for x86_64",
+            "id": "479"
+          }
+        ],
+        "cores_per_socket": 1,
+        "system_memory_bytes": 3836579840,
+        "number_of_cpus": 1,
+        "number_of_sockets": 1
+      },
+      "cpu_socket(s)": "1"
+    }
+  ]
+}
+----
```

Comment:
I'd move the example between lines 12-14. I cannot leave you an explicit suggestion for this obviously :)

Reply:
Fixed